python/default.nix

```diff
@@ -1,42 +1,35 @@
-let
```

Comment:
Won't this make nix-shell not work in the c/ and python/ directories?

Reply:
Not sure I like breaking it out into `package.nix`  but this does make it work the same way in all circumstances.